Gemfile

```diff
@@ -3,6 +3,8 @@
 source 'https://rubygems.org'
 gemspec
 
+gem 'lru_redux'
```

Comment:
This gem is used by other logstash filters so should be reasonable to use in our own as well.